src/Helper/OutputResultConverter.php

```diff
@@ -82,14 +83,26 @@ public static function convertOutputsToMetrics(array $outputs, Backend $backend)
     {
         $jobMetrics = new JobMetrics();
 
-        $sum = 0;
+        $inputTablesCompressedBytesSum = 0;
+        $outputTablesCompressedBytesSum = 0;
         foreach ($outputs as $output) {
             $inputTableResult = $output->getInputTableResult();
             if ($inputTableResult) {
                 if ($inputTableResult->getMetrics()) {
                     foreach ($inputTableResult->getMetrics()->getTableMetrics() as $tableMetric) {
-                        /** @var TableMetrics $tableMetric */
-                        $sum += $tableMetric->getCompressedBytes();
+                        /** @var InputTableMetrics $tableMetric */
+                        $inputTablesCompressedBytesSum += $tableMetric->getCompressedBytes();
+                    }
+                }
+            }
+
+            $outputTableResult = $output->getOutputTableResult();
+            if ($outputTableResult) {
+                $outputMetrics = $outputTableResult->getMetrics();
+                if ($outputMetrics) {
+                    foreach ($outputMetrics->getTableMetrics() as $tableMetric) {
+                        /** @var OutputTableMetrics $tableMetric */
```

Comment:
ten typehint by tu podle mě neměl být potřeba jestli to má nastavenej správnej (jakože si myslím, že má)